chains/kyvetestnet/metadata.yaml

```diff
@@ -33,7 +33,7 @@ nativeToken:
   denom: tkyve
   name: KYVE
   symbol: KYVE
```

Comment:
kinda wonder if this should've been named something to do with kaon, but nbd

Reply:
i think the network is still called kyve kaon so idm keeping the nativetoken as kyve (which is what the explorer shows)